TacTok/options.py

```diff
@@ -61,11 +61,14 @@ def parse_args():
     # The local variable vocabulary
     if opts.include_locals:
         vocab += list(pickle.load(open(opts.local_vocab_file, 'rb')).keys())
-        vocab += ['<unk-local>']
+        if not opts.merge_vocab:
+            vocab += ['<unk-local>']
 
-    if opts.include_paths:
+    # The path vocabulary
+    if opts.include_paths and not opts.merge_vocab:
```

Comment:
This "not" is sketch and probably shouldn't be here. @HazardousPeach we need to look at this at some point